### ISSUE TITLE
trainimgs and valimgs not defined

### ISSUE THREAD

**Issue by KronosTheLate** [FluxML/model-zoo#322]
From the 60-minute blitz:
```julia
using Statistics
using Flux, Flux.Optimise
using Images: channelview
using Metalhead
using Metalhead: trainimgs, valimgs
using Images.ImageCore
using Flux: onehotbatch, onecold, flatten
using Base.Iterators: partition
```

However, I get the following error:
![image](https://user-images.githubusercontent.com/61620837/142859515-04136b16-6153-4efe-979d-a9e5a2aaaef6.png)

So the example is not functional.

**Comment by DhairyaLGandhi** [comment]
I think this is broken with the current Metalhead release. Could you try with `Metalhead@0.5`

**Comment by KronosTheLate** [comment]
I just tried with the correct enviroment activated and instantiated. So the example works. But it should still be updated, right? To lower the barrier of entry - I am not sure if the people finding their way to the introduction all are comfortable with cloning Git-repos and juggling new enviroments in Julia.

**Comment by DhairyaLGandhi** [comment]
Absolutely. I think this should be something we get back into metalhead eventually.

**Comment by KronosTheLate** [comment]
Cool. Would it be too much to make an issue over there, asking for it to be added again, with a reference to this conversation?

**Comment by DhairyaLGandhi** [comment]
Can do that. Ideally we wouldn't want to add more deps, but have an simple implementation for data preprocessing etc like we used to.

**Comment by darsnack** [comment]
These functions already exist in MLDatasets.jl. Metalhead.jl was intentionally stripped down to just the models to reduce feature duplication across packages. Instead, the tutorial should be updated to use MLDatasets.jl.

**Comment by casasgomezuribarri** [comment]
`using MLDatasets: trainimgs` and `using MLDatasets: valimgs` do not work either

![image](https://user-images.githubusercontent.com/71462845/145816189-b9c065da-1a64-4a1b-8a4b-1932b37af43d.png)

Also, running the line `Metalhead.download(Metalhead.CIFAR10)` gives an error too:

![image](https://user-images.githubusercontent.com/71462845/145816371-338201c5-01fa-4cf6-ad19-a412c2c0ea67.png)

(Running `Metalhead.download(CIFAR10)` or `Metalhead.download(MLDatasets.CIFAR10)` returns an error 

> ERROR: LoadError: MethodError: no method matching download(::Module)

How can I go through the tutorial?

**Comment by DhairyaLGandhi** [comment]
Use Metalhead v0.5

**Comment by casasgomezuribarri** [comment]
Doing 
```julia
> ]add Metalhead@0.5
> using Metalhead: trainimgs 
```
returns the same error (`Pkg.status` says I'm using Metalhead v0.5.3)

Doing 
```julia
> ]add Metalhead@0.5.0
```

returns an incompatibility error with Flux
```julia
ERROR: Unsatisfiable requirements detected for package Flux [587475ba]:
 Flux [587475ba] log:
 ├─possible versions are: 0.4.1-0.12.8 or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions 0.4.1-0.12.8
 ├─restricted by compatibility requirements with Metalhead [dbeba491] to versions: 0.10.0-0.10.4
 │ └─Metalhead [dbeba491] log:
 │   ├─possible versions are: 0.3.0-0.6.1 or uninstalled
 │   └─restricted to versions 0.5.0 by an explicit requirement, leaving only versions 0.5.0
 └─restricted by compatibility requirements with CuArrays [3a865a2d] to versions: [0.4.1-0.8.3, 0.11.0-0.12.8] or uninstalled — no versions left
   └─CuArrays [3a865a2d] log:
     ├─possible versions are: 0.2.1-2.2.2 or uninstalled
     └─restricted by julia compatibility requirements to versions: uninstalled
```

`]rm Flux` does not solve the problem

![image](https://user-images.githubusercontent.com/71462845/145819278-9a00b4ac-1f6d-4674-af77-1bb6750ae1fb.png)

**Comment by ToucheSir** [comment]
Simply changing the package name will not work. Please see the updated tutorial at https://fluxml.ai/tutorials/2020/09/15/deep-learning-flux.html.

**Comment by darsnack** [comment]
> using MLDatasets: trainimgs and using MLDatasets: valimgs do not work either

This should be `using Metalhead.CIFAR10: traintensor, testtensor` (or `traindata`/`testdata` to get the images + labels. The updated tutorial that @ToucheSir linked reflects this.
